iphone/Classes/MediaModule.m

```diff
@@ -670,28 +669,71 @@ - (void)requestAudioRecorderPermissions:(id)args
 
 - (void)startMicrophoneMonitor:(id)args
 {
-  [[SCListener sharedListener] listen];
+  if (_microphoneRecorder != nil) {
+    DebugLog(@"[ERROR] Trying to start new microphone monitoring while the old one is still active. Call \"stopMicrophoneMonitor()\" before and try again.");
+    return;
+  }
+
+  NSURL *url = [NSURL fileURLWithPath:@"/dev/null"];
```

Comment:
Is value @"/dev/null" is correct for path or we should get path using [NSFileMange defaultManage] api to get path?